src/Components/AddComment.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import postComment from "../../utils/add-comment";
+import getCommentsById from "../../utils/get-comments-by-id";
+
+function AddComment(props) {
+  const { articleId } = props;
+  const [username, setUsername] = useState("");
+  const [comment, setComment] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(articleId, comment, username).then((response) => {
+      if (response) {
+        setUsername("");
+        setComment("");
+        getCommentsById();
```

Comment:
This line (17) isn't doing anything currently as you're not using the fetched comments to do anything or update the state of the comments 🔍